app/helpers/alchemy/admin/pages_helper.rb

```diff
@@ -1,29 +1,9 @@
+# coding: utf-8
```

Comment:
Yes, @jorrizza please remove this. If you have an editor that automatically inserts this, please disable it for Alchemy repo. Thanks